eap/eap71-tx-recovery-s2i.json

```diff
@@ -158,6 +165,47 @@
         }
     ],
     "objects": [
+        {
+            "apiVersion": "v1",
+            "kind": "Role",
+            "metadata": {
+                "name": "listing-pod-role"
+            },
+            "rules": [
+                {
+                    "apiGroups": null,
+                    "attributeRestrictions": null,
+                    "resources": ["pods", "pods/log"],
+                    "verbs": ["list", "get"]
+                }
+            ]
+        },
+        {
+            "apiVersion": "v1",
+            "kind": "ServiceAccount",
+            "metadata": {
+                "name": "listing-pod"
```

Comment:
Using a static name in the template will cause problems on subsequent invocations, because the service account already exists.